.github/workflows/tests.yml

```diff
@@ -1,5 +1,8 @@
 name: Testing BE
-on: pull_request
+on: 
+  pull_request:
+    paths:
+      - '**.py'
```

Comment:
I'd recommend to not skip Python unittests in case there were no changes in Python. Other things can change (environment variables, for example), unittests execution doesn't take too long, so there is a sense to run it on each deployment.

But it's up to you. I'm approving PR.